### PR TITLE
Add MyImages to user Profile

### PR DESCRIPTION
**Pull request by bladehan2020**
Fix the original permission issues and all good in the testing.

### CODE REVIEW

File: app/controllers/uploaded_images_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        @title = @user.display_name + "'s uploaded images"
+      end
+    end
+    @images = UploadedImage.get_uploaded_images(current_user)
+  end
+
+  def destroy_multiple
+    if params[:deleted_img_ids]
+      for id in params[:deleted_img_ids] do
+        uri = URI('http://noter-backend:3001/api/v0.1/images/' + id.to_s + '/')
```
Comment (rkversion) [inline app/controllers/uploaded_images_controller.rb:24]: Is this guaranteed to always work?
Wouldn't it be better to have a config parameter with API URI?

File: app/views/uploaded_images/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <% if @user %>
+      <%= user_image @user %>
+    <% end %>
+    <h1><%= h(@title) %></h1>
+
+    <ul class='secondary-actions clearfix'>
+      <% if @user %>
+        <% if @user == current_user %>
+          <div>
+            <li><%= link_to image_tag("new.png", :class => "small_icon", :border => 0) + "Upload", "http://localhost/nf/", :title => t(".new_title") %></li>
```
Comment (rkversion) [inline app/views/uploaded_images/index.html.erb:12]: localhost?
where is this supposed to run?

**Comment by bladehan2020** [comment]
Wops, forgot updating this.
The  idea for the editor to talk to the noter-backend is through the local
network,  so here should be the same as calling other noter-backend apis to
use the name of the noter-backend service.  Let me fix this.

**Comment by bladehan2020** [comment]
Yes, it would be better to put them into a config. let me look into that.

**Pull request merged**